athlinks-ontology.ttl

```diff
@@ -6,542 +6,680 @@
+
+:SwimEvent a owl:Class .
+# 
+# 
+# 
+# #################################################################
+# #
+# #    Individuals
+# #
+# #################################################################
+# 
+# 
+# http://adp.athlinks.com/schema/AdventureEvent
+
+:AdventureEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/AquabikeEvent
+
+:AquabikeEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/AquathlonEvent
+
+:AquathlonEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/CyclingEvent
+
+:CyclingEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/DuathlonEvent
+
+:DuathlonEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/MountainBikeEvent
+
+:MountainBikeEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/MultisportHalfIronmanEvent
+
+:MultisportHalfIronmanEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/MultisportIronManEvent
+
+:MultisportIronManEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/MultisportOlympicEvent
+
+:MultisportOlympicEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/MultisportSprintEvent
+
+:MultisportSprintEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/ObstacleEvent
+
+:ObstacleEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/OtherEvent
+
+:OtherEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/Run15kToHalfMarathonEvent
+
+:Run15kToHalfMarathonEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/Run5kTo15kEvent
+
+:Run5kTo15kEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/RunHalfToFullMarathonEvent
+
+:RunHalfToFullMarathonEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/RunMarathonEvent
+
+:RunMarathonEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/RunUltraEvent
+
+:RunUltraEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/RunUpTo5kEvent
+
+:RunUpTo5kEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/SnowEvent
+
+:SnowEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# http://adp.athlinks.com/schema/SwimEvent
+
+:SwimEvent a owl:NamedIndividual , :AthleticEventCategory .
+# 
+# 
+# 
+# #################################################################
+# #
+# #    Annotations
+# #
+# #################################################################
+# 
+# 
+
+:AdventureEvent rdfs:label "Adventure Event"@en ;
```

Comment:
These annotations should be located with the original declaration of the subject (at line 102, or better line 408), no?

Reply:
Yes, but protege generates the file automatically.